warble/functions.cc

```diff
@@ -98,8 +124,7 @@ bool Warble(Database* db, Any req, Any* rep) {
   }
 
   // Adds hashtag to list of hashtag if found
-  Hashtag(db, warble_id, w_as_string);
-
+  Hashtag(db, warble_id, text);
```

Comment:
Is this incomplete, or no longer needed?

Reply:
Here I'm trying to add the warble id of the new warble in to kvstore as (key: hashtag, value: {ids}). I just changed w_as_string to text because I realised after a painfully long debugging time that w_as_string wasn't the content of the warble. The functionality is complete though.

Reply:
Side note, do we need to add the professor and TAs as reviewers for each PR